src/main/kotlin/com.bugsnag.android.gradle/Abi.kt

```diff
@@ -0,0 +1,41 @@
+package com.bugsnag.android.gradle
+
+enum class Abi(val abiName: String, val toolchainPrefix: String, val objdumpPrefix: String) {
+    ARMEABI(
+        "armeabi",
+        "arm-linux-androideabi",
+        "arm-linux-androideabi"
+    ),
+    ARMEABI_V7A(
+        "armeabi-v7a",
+        "arm-linux-androideabi",
+        "arm-linux-androideabi"
+    ),
+    ARM64_V8A(
+        "arm64-v8a",
+        "aarch64-linux-android",
+        "aarch64-linux-android"
+    ),
+    X86(
+        "x86",
+        "x86",
+        "i686-linux-android"
+    ),
+    X86_64(
+        "x86_64",
+        "x86_64",
+        "x86_64-linux-android"
+    );
+
+    companion object {
+        fun findByName(abiName: String): Abi? {
```

Comment:
I'm not really familiar with Kotlin, but I think if this was [`@JvmStatic`](https://kotlinlang.org/docs/reference/java-to-kotlin-interop.html#static-methods) you could avoid having to change how `BugsnagUploadNdkTask` calls this; is that worth doing?

Reply:
I tried `@JvmStatic` and it seems that Groovy is unable to find the correct method on `Abi` without the small change in `BugsnagUploadNdkTask`. As a rewrite of `BugsnagUploadNdkTask` in Kotlin is imminent I'm assuming this will be ok for now.

Reply:
Makes sense 👍